maintainer/channels.py

```diff
@@ -89,12 +88,12 @@ def delete_parents(self):
             channel = channels_data[channel_id]
             if channel["channel_topic"] != 'protected' and channel["channel_topic"] == 'quarantine' and int(channel["seconds_empty"]) > self.qtime:
                 self._requests_delete('channels/{}'.format(channel_id))
-            return logger.log("deleted channels {}".format(channel_id))
+            logging.debug("deleted channels {}".format(channel_id))
 
     def delete_children(self):
         '''
         Method is used for deleting channels that are having children status.
-        Output: deleted children channel
+        Output: log with information about deleted children channel
```

Comment:
fix this, there is no return/yield statement, how the method can return something?